File: 310971/Khulup/task_350_redis/publisher/app/models/issue.py
from pydantic import BaseModel, ConfigDict, Field
from datetime import datetime

class Issue(BaseModel):
    model_config = ConfigDict(from_attributes=True, populate_by_name=True)
    id: int
    userId: int = Field(alias="user_id")
    title: str
    content: str
    created: datetime
    modified: datetime

File: 351002/Mikhalkov/Distcomp.Domain/Models/Issue.cs
namespace Distcomp.Domain.Models
{
    public class Issue
    {
        public long Id { get; set; }
        public long UserId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Content { get; set; } = string.Empty;
        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }

        public List<long> MarkerIds { get; set; } = new();
    }
}